db/migrate/20230330224511_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :commentscounter
+      t.integer :likescounter
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] It would be nice if you could create a reference for the author_id in this file as this will create author_id and a corresponding index.

Reply:
**Hello dear Reviewer,** 

Please notice that I created a separate migration for the association User/Post and renamed the user_id  to auther_id inside the Post Table